doc/src/gotchas.rst

```diff
@@ -270,6 +270,33 @@ trick for getting tab completion in the regular Python console.
     See also `What is the best way to create symbols?
     <https://github.com/sympy/sympy/wiki/Faq>`_ in the FAQ.
 
+.. _symbols:
+
+Functions
+---------
+
+A function like ``f(x)`` can be created by defining the Function
+and the variable:
+
+  >>> from sympy import Function
+  >>> f = Function('f')
+  >>> x = Symbol('x')
+  >>> f(x)
+  f(x)
+
+If you assign ``f(x)`` to a Python variable `f` you will lose your
+ability to copy and paste that function or to create a function
```

Comment:
What do you mean by "copy and paste"?

Reply:
@mohitacecode Please clarify on this.

Reply:
Actually This was suggested here (https://github.com/sympy/sympy/issues/16836#issue-443970391) but I have changed it can you see if it seems ok to you. Here (https://github.com/sympy/sympy/pull/18646#issuecomment-585807833)